understanding/20/non-text-content.html

```diff
@@ -8,6 +8,15 @@
 <body>
    <h1>Understanding Non-text Content</h1>
    
+   <section id="brief">
+		<h2>In brief</h2>
+		<dl>
+		<dt>Objective</dt><dd>Non-text information is available to more people</dd>
```

Comment:
```suggestion
		<dt>Objective</dt><dd>Information about images and other non-text content is available to more people</dd>
```

Reply:
Agreed that what there needs improvement.   
  
I wonder about reducing the repetition and focusing on examples.

from:  
"Non-text information is available to more people  
Information about images and other non-text content is available to more people"  
  
to:  
"Non-text information is available to more people  
Images, charts, audio/video are non-text content that convey meaning that many people can not observe."

Reply:
In reality, the primary use case for 1.1.1 tends to be for images, so it was felt calling that out directly makes sense (and also helps convey what "non-text content" means, to someone unfamiliar with the term).  As well, there are many SCs just covering audio/video, so I'm concerned that adding those may actually increase confusion, not reduce it.